packages/react/src/toast/Toast.css.ts

```diff
@@ -119,6 +119,11 @@ export const root = recipe({
           [accentColorVar]: theme.colors["red.300"],
         },
       }),
+      informational: style({
```

Comment:
this should be `information`
```suggestion
      information: style({
```